### PR TITLE
fix(css): fix css for planner datatable and quick add

### PR DESCRIPTION
**Pull request by SMahil**
This PR solves:
1. fix wi title and last column width
2. Adding visibility to quick add icon
3. remove extra classes
4. fix css for experimental view (NOTE: THIS IS TEMPORARY FIX TILL GA AS IT INCLUDES A LOT OF HACKS AND TIME.)

### CODE REVIEW

File: src/app/components/planner-list/planner-list.component.ts [hunk 1]
```diff
@@ -181,7 +181,7 @@ export class PlannerListComponent implements OnInit, AfterViewChecked, OnDestroy
     this.loggedIn = this.auth.isLoggedIn();
 
     // Cookie for datatableColumn config
-    if(!this.cookieService.getCookie(datatableColumn.length).status) {
+    if(this.cookieService.getCookie(datatableColumn.length).status) {
```
Comment (vikram-raj) [inline src/app/components/planner-list/planner-list.component.ts]: Why you change this condition?

Comment (SMahil) [reply src/app/components/planner-list/planner-list.component.ts]: that was for testing at that time. I missed to undo the change

**Comment by SMahil** [comment]
inline quick add
screen shot 2018-01-17 at 3 37 36 pm
screen shot 2018-01-17 at 3 39 09 pm

**Comment by SMahil** [comment]
wi title width and last column width
screen shot 2018-01-17 at 3 39 36 pm
screen shot 2018-01-17 at 3 39 48 pm
screen shot 2018-01-17 at 3 42 30 pm
screen shot 2018-01-17 at 3 42 42 pm

**Comment by SMahil** [comment]
screen shot 2018-01-17 at 3 43 35 pm
screen shot 2018-01-17 at 3 43 24 pm

**Comment by sahil143** [comment]
@SMahil On Logout, clicking on a disabled quickAdd icon changes its color from grey to blue.
screen shot 2018-01-17 at 3 45 11 pm

**Comment by SMahil** [comment]
@sahil143 I didn't get this issue. Neither on local nor on snapshot

screen shot 2018-01-17 at 4 34 39 pm

**Comment by nimishamukherjee** [comment]
@SMahil @sahil143 @sanbornsen how do scroll the table horizontally.
I know the wireframe gives an infinite table but don't we need a fixed height table so that we can see the horizontal scrollbar.

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2405-7 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2405-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
LGTM

**Comment by Raunak1203** [comment]
if we move all 4 attribute from displayed to available and then scroll horizontally there is extra white space in the right after kebab menu

**Comment by vikram-raj** [comment]
@Raunak1203 ak1203  good catch, but this issue not related to this PR and it is related to ngx-datatable will fix this issue in separate PR. Can you please create an issue on OSIO so that we can track this issue. and ``` ngx-datatable``` label to it.

**Comment by rgarg1** [comment]
@SMahil @vikram-raj 

Notice the ugly gray color padding near the quick add:

unneccessary padding

This is seen when th euser navigates away from __Plan__ to __Create__ screen and then back to __Plan__.

**Comment by SMahil** [comment]
@rgarg1 This is the existing issue where in "create" extra padding has been added. This issue has to resolve from there

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2405-13 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2405-fabric8-planner.badger.fabric8.io

**Pull request merged**